packages/core/src/agent/TransportService.ts

```diff
@@ -39,7 +39,7 @@ interface TransportSessionTable {
 export interface TransportSession {
   id: string
   type: string
-  keys?: EnvelopeKeys
+  keys?: PackMessageParams
```

Comment:
Is this still keys now? For didcomm v2 it will be the dids right?
Also I'm not sure if this type is the right type to use for the transport keys. This is also used before packing which means it doesn't include the exact key, which it does for v1. I think we should come up with a type that work for both v1 and v2 and don't try to reuse when not applicable